sycl/test-e2e/ESIMD/unified_memory_api/Inputs/block_store.hpp

```diff
@@ -11,13 +11,33 @@
 using namespace sycl;
 using namespace sycl::ext::intel::esimd;
 
+// Returns true iff verification is passed.
+template <typename T>
+bool verify(T Out_val, const T *Out, size_t Size, int N, bool UseMask) {
```

Comment:
nit: it is mix of Clang vars naming and C++ naming in one variable:
```suggestion
bool verify(T OutVal, const T *Out, size_t Size, int N, bool UseMask) {
```

Reply:
fixed, thx